Nodejs/Product/Nodejs/NodejsPackage.cs

```diff
@@ -251,6 +251,11 @@ internal sealed partial class NodejsPackage : CommonPackage {
             Environment.SetEnvironmentVariable(NodejsConstants.NodeToolsProcessIdEnvironmentVariable, Process.GetCurrentProcess().Id.ToString());
         }
 
+        public static LANGPREFERENCES3[] GetNodejsLanguagePreferencesFromTypeScript() {
+            IVsTextManager4 textMgr = (IVsTextManager4)Instance.GetService(typeof(SVsTextManager));
+            return GetNodejsLanguagePreferencesFromTypeScript(textMgr);
+        }
```

Comment:
This is unnecessary - we should be able to use NodejsPackage.Instance.LangPrefs to access this property.